### ISSUE TITLE
[META] Add issue templates

### ISSUE THREAD

**Issue by CosminPerRam** [ValveSoftware/Source-1-Games#5781]
# About
Github has [issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository), by their name, are customizable templates that users can use (or should, just as a suggested thing) when making an issue or even for pull requests.

# Why?
Seeing the recent issues wave because of the Tf2 64-bit update, this could greatly help people to make a readable, insightful and consistent report, as a lot of them are just one-liners and are not that helpful.

Although the requirements for a well-written issue are already in the readme this imposes the rules a lot more.

Multiple templates can be done and they support things such as dropdowns and checkboxes.

If there is interest into adding this I could make some template suggestions that I think fits around here, would be glad to hear some thoughts about this as I think the repository could benefit a lot for this.

**Comment by CosminPerRam** [comment]
Providing more context about some examples:
To see some realistic examples, over at [facepunch/sbox-issues](https://github.com/Facepunch/sbox-issues) they do have some templates:
![image](https://github.com/ValveSoftware/Source-1-Games/assets/7972857/72f2b323-be7d-4203-a20b-7e694dc7a9bb)
An example of the bug report one:
![image](https://github.com/ValveSoftware/Source-1-Games/assets/7972857/4c181efd-dd4a-4fc1-973b-60872d133df0)

**Comment by CosminPerRam** [comment]
Even more: you can also set `labels` for each issue, so users dont have to include these in the title (such as `[TF2]`, `[BUG]`, `[LINUX]`, `[SUGGESTION]` and so on).

**Comment by ArtKitCat** [comment]
yeah i have to agree on this, there's over 3000+ issues currently and some dating as far back as 2012. There's no way over 3k issues are being kept up to date and actively worked on.

**Comment by TheBoctor** [comment]
This would help a lot, especially with the tracker being linked in the [patch notes](https://www.teamfortress.com/post.php?id=220436) for the 64-bit branch's mainlining. It seems like a lot of new users are now coming here to report client crashes, but many are also new to trackers, and so they may forget to include OS/machine details, let alone logs or dumps.

**Comment by CosminPerRam** [comment]
@kisak-valve hey, hope you don't mind the ping, any thoughts on this coming from an official standpoint? seems like many agree on adding this.
